### DISCUSSION TITLE
나의 발자취 | kciter.so

### DISCUSSION THREAD

**Discussion by giscus** [kciter/kciter.github.io#26] [Announcements]
# 나의 발자취 | kciter.so

삶의 지도 개발 밥을 먹은 지도 벌써 10년이 됐습니다. 저에게 개발이라는 행위가 주는 즐거움은 아직도 유효하지만, 미지의 것을 탐구하려는 열정은 식어버린 것 같습니다. 신경 쓸 것이 많다, 몸이 옛날 같지 않다 같은 핑계만 늘어나고 있는 요즘, 이…

https://kciter.so/posts/retrace-my-life

**Comment by ibcylon** [comment]
🐸
